docs/coordinates/index.rst

```diff
@@ -521,7 +521,6 @@ coordinate systems implemented here include:
 
 Built-in Frame Classes
 ======================
-
```

Comment:
Why removing the blank line here? I guess if RTD doesn't complain, it is fine.

Reply:
Uh, weird, not sure why this happened - my text editor is getting more chaotic!